docs/Guide/preconditions/command-cooldown.mdx

```diff
@@ -1,6 +1,278 @@
 ---
-sidebar_position: 0
+sidebar_position: 2
 title: Configuring command cooldowns
 ---
 
-## TODO
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Cooldowns are of vital importance for many bots to avoid spam, API ratelimits, etc. Luckily, Sapphire makes it easy to
```

Comment:
```suggestion
Cooldowns are of vital importance for many bots to avoid excessive command usage, API ratelimits, and so on. Luckily, Sapphire makes it easy to
```